Doc/library/code.rst

```diff
@@ -18,9 +18,9 @@ build applications which provide an interactive interpreter prompt.
    This class deals with parsing and interpreter state (the user's namespace); it
    does not deal with input buffering or prompting or input file naming (the
    filename is always passed in explicitly). The optional *locals* argument
-   specifies the dictionary in which code will be executed; it defaults to a newly
-   created dictionary with key ``'__name__'`` set to ``'__console__'`` and key
-   ``'__doc__'`` set to ``None``.
+   specifies the mapping namespace in which code will be executed; it defaults
```

Comment:
If you'll entertain some pedantry, I'd say that "mapping namespace" is not a common term of art. Rather I'd say that "The optional locals argument specifies a mapping to use as the namespace in which the code will be executed."

Reply:
That wording is consistent with the eval() doc in functions.rst

Reply:
I think I used "mapping namespace" in a couple of other places to. I agree it's an odd phrase, so I'll hunt them down and replace them with something better (such as the `eval` wording)

Reply:
Turned out this was the only case, so I changed it as suggested.